src/libs/getCurrentPosition/index.android.ts

```diff
@@ -15,9 +15,8 @@ const getCurrentPosition: GetCurrentPosition = (success, error, config) => {
     // Prompt's the user to enable geolocation permission with yes/no options
     // If the user selects yes, then this module would enable the native system location
     // Otherwise if user selects no, or we have an issue displaying the prompt, it will return an error
-    RNAndroidLocationEnabler.promptForEnableLocationIfNeeded({
+    promptForEnableLocationIfNeeded({
         interval: 2000, // This updates location after every 2 seconds (required prop). We don't depend on this as we only use the location once.
-        fastInterval: 1, // The shortest time (1 ms) our app is willing to wait for location update. Passing 0 ms short's the internal ternary condition of library to default value.
     })
```

Comment:
It means that we will use the default value (0 ms) for the `fastInterval` param. Why do we need to do that?

Reply:
This param doesn't exist in version `2.0` of this library: https://github.com/Richou/react-native-android-location-enabler/blob/c3d248c410f9a6174d3f5f546cc47a207253716f/src/index.tsx#L22-L25